compress_jointcal_cfht_test_data.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
```

Comment:
Currently we do not distribute astropy with eups so it's in the standard location.